plugins/react-native.js

```diff
@@ -69,7 +69,12 @@ function reactNativePlugin(Raven) {
         }
     });
 
-    ErrorUtils.setGlobalHandler(Raven.captureException.bind(Raven));
+    var defaultHandler = (ErrorUtils.getGlobalHandler && ErrorUtils.getGlobalHandler())
+     || ErrorUtils._globalHandler;
+    ErrorUtils.setGlobalHandler(function(...args){
+      defaultHandler(...args);
```

Comment:
I'd preserve the value of `this` in case that's a factor later: `defaultHandler.apply(this, arguments)`